umbral/pre.py

```diff
@@ -46,6 +47,13 @@ def __init__(self, message: str, offending_cfrags: List[CapsuleFrag]) -> None:
         self.offending_cfrags = offending_cfrags
 
 
+class UmbralDecryptionError(GenericUmbralError):
+    def __init__(self) -> None:
+        super().__init__("Decryption of ciphertext failed: "
```

Comment:
Not sure if this is really necessary here, can't you just do a `return` if you want this?

Reply:
Not sure what you mean here 🤔